### PR TITLE
Add test for nullification, fix datepicker nullification

### PR DESCRIPTION
**Pull request by rkuykendall**

### CODE REVIEW

File: test/types/types.test.ts [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import { Tester } from '@mighty-justice/tester';
 import { EMPTY_FIELD } from '@mighty-justice/utils';
 
-import { Card, fillInFieldConfig, FormCard, TYPES } from '../../src';
+import { Card, fillInFieldConfig, FormCard, IFieldConfig, TYPES } from '../../src';
 import { TYPE_GENERATORS, valueRenderPairs } from '../factories';
+import { IValue } from '../../src/props';
```
Comment (ekatsuta) [inline test/types/types.test.ts]: is this similar to importing IInjected for possible future use case? IValue not used anywhere else

Comment (rkuykendall) [reply test/types/types.test.ts]: NOPE! This is a mistake you found :)

File: test/types/types.test.ts [hunk 2]
```diff
@@ -48,6 +48,22 @@ Object.keys(TYPE_GENERATORS).forEach(type => {
...
+      const emptyValue = (() => {
+          if (fieldConfig.type === 'checkbox') { return false; }
+          return fieldConfig.nullify ? null : '';
+        })()
+        , onSave = jest.fn()
+        , props = { fieldSets, onSave }
+        , expectModel = { [fieldConfig.field]: emptyValue }
+        ;
+
+      const tester = await new Tester(FormCard, { props }).mount();
```
Comment (SHA-4) [inline test/types/types.test.ts]: This works because it mounts quickly, but shouldn't you pass `async: true`?

File: test/types/types.test.ts [hunk 3]
```diff
@@ -48,6 +48,22 @@ Object.keys(TYPE_GENERATORS).forEach(type => {
       expect(onSave).toHaveBeenCalledWith(model);
     });
 
+    it('Has correct empty values', async () => {
+      const emptyValue = (() => {
```
Comment (SHA-4) [inline test/types/types.test.ts]: I would use a named function for this that takes a `fieldConfig` prop. I can imagine this being useful in other tests

**Pull request merged**